src/main/java/org/opentripplanner/client/parameters/TripPlanParametersBuilder.java

```diff
@@ -9,9 +9,10 @@
 import org.opentripplanner.client.parameters.TripPlanParameters.SearchDirection;
 
 public class TripPlanParametersBuilder {
-
   private Coordinate from;
+  private String fromPlace;
   private Coordinate to;
+  private String toPlace;
```

Comment:
Can you remove from/to and replace it with an interface that has two implementations:

- StopId
- Coordinate

Both types have a method to convert their value to a string that is suitable for `fromPlace`/`toPlace`.

The format for coordinate for would look like this: `48.24207, -122.4605::48.242069284370444,-122.46050285644506`

This would allow you to remove a few things in this PR.